isis/src/tgo/apps/tgocassismos/main.cpp

```diff
@@ -184,17 +186,27 @@ void IsisMain() {
         observationId =  archiveGroup["ObservationId"][0];
         startTime =      instGroup["StartTime"][0];
         startClock =     instGroup["SpacecraftClockStartCount"][0];
+        stopTime   =     instGroup["StartTime"][0]; 
+        exposureDuration  = instGroup["ExposureDuration"][0]; 
       }
       else {
-        QString testStartTime = instGroup["StartTime"][0];
-        if (testStartTime < startTime) {
-          startTime = testStartTime;
+        iTime testStartTime = iTime(instGroup["StartTime"][0]);
+        iTime testStopTime = iTime(instGroup["StartTime"][0]);
```

Comment:
Is this meant to be "StopTime" by chance?

Reply:
Same as above.

Reply:
Ah! Makes sense!